ui/src/components/home/VisionMissionSection.js

```diff
@@ -0,0 +1,412 @@
+  VStack,
+  Text,
+  Spacer,
+  Link,
+  Image,
+  Fade,
+  Container,
+} from '@chakra-ui/react';
+import { ArrowForwardIcon } from '@chakra-ui/icons';
+import { InView } from 'react-intersection-observer';
+const VisionMissionSection = () => {
+  return (
+    <InView rootMargin="-200px" triggerOnce={true}>
+      {({ inView, ref }) => (
+        <Fade transition={{ enter: { duration: 1 } }} in={inView}>
+          <Container
+            maxW="container.xl"
+            py={10}
+            ref={ref}
+            bgImage={`url('${process.env.PUBLIC_URL}/images/home/hk-green.png')`}
+            bgPos="center right"
+            bgSize="contain"
+            bgRepeat="no-repeat"
+          >
+            <VStack>
+              <Box w="100%" alignItems="flex-start" paddingBottom={'1em'}>
+                <Box w="20%" position="relative">
+                  <Box
+                    w="100%"
+                    position="absolute"
+                    h="100%"
+                    bgPos="45% 50%"
+                    bgSize="40%"
+                    bgRepeat="no-repeat"
+                    bgImage={`url('${process.env.PUBLIC_URL}/images/home/vector-green-1.svg')`}
+                  ></Box>
+                  <Box
+                    w="100%"
+                    position="absolute"
+                    h="107%"
+                    bgPos="30% 100%"
+                    bgSize="50%"
+                    bgRepeat="no-repeat"
+                    bgImage={`url('${process.env.PUBLIC_URL}/images/home/vector-green-2.svg')`}
+                  ></Box>
+                  <Heading
+                    color="#0C0C20"
+                    fontFamily="DMSerifDisplay_Italic, serif"
+                    fontWeight="400"
+                    lineHeight="1"
+                    wordBreak="break-word"
+                  >
+                    Our Vision<br></br> & Mission
+                  </Heading>
+                </Box>
+              </Box>
+              <Flex>
+                <VStack
+                  borderRadius="0.5rem"
+                  w="49.5%"
+                  border="1px solid"
+                  borderColor="green.400"
+                  p="1em"
+                >
+                  <Box borderRadius="1.875em" bgColor="white" m="auto" px="1em">
+                    <Text
+                      color="#0C0C20"
+                      fontSize="20px"
+                      fontFamily="Manrope"
+                      fontWeight="700"
+                      textTransform="uppercase"
+                      lineHeight="20.6px"
+                      letterSpacing="4px"
+                      wordBreak="break-word"
+                    >
+                      VISION
+                    </Text>
+                  </Box>
+                  <Box>
+                    <Text>
+                      <Text display="inline">
+                        <Text
+                          color="#4A6EEB"
+                          fontSize="20px"
```

Comment:
i'm pretty sure you can move the "fontSize" "fontWeight" "lineHeight" to the parent element and it would still apply the effect for the child

Reply:
also 'fontSize' is usually better to use rem and then try to eyeball match with the design. makes it better for different device sizes i think. a font size of 20 in Figma was a match with `1.4rem` for me. So I think this current fontSize is too small. You can somewhat do mathematical conversion using this reference point to match font sizes on the design with a bit of tuning (rounding up/down)